.github/workflows/e2e.yml

```diff
@@ -1,48 +1,100 @@
-      DOCKER_COMPOSE_PATH: './docker/docker-compose.yml'
+      DATA_IMAGE_NAME: '${{ vars.DB_NAME }}-data'
+    outputs:
+      DATA_MANIFEST_ID: ${{ steps.get-ids-from-package-json.outputs.DATA_MANIFEST_ID }}
+      DATA_VERSION: ${{ steps.get-ids-from-package-json.outputs.DATA_VERSION }}
+      DATA_IMAGE_PATH: ${{ steps.get-ids-from-package-json.outputs.DATA_IMAGE_PATH }}
+      IMAGE_EXISTS: ${{ steps.check-container-repo.outputs.IMAGE_EXISTS }}
     steps:
       - uses: actions/checkout@v4
 
       - name: get data image tag, data manifest id, and data version
+        id: get-ids-from-package-json
         env:
           PACKAGE_JSON_PATH: 'package.json'
-          DATA_IMAGE_NAME: '${{ vars.DB_NAME }}/data'
         run: |
           DATA_MANIFEST_ID=$(jq -r '."data-file"' "${{ env.PACKAGE_JSON_PATH }}")
-          echo "DATA_MANIFEST_ID=${DATA_MANIFEST_ID}" >> "$GITHUB_ENV"
+          echo "DATA_MANIFEST_ID=${DATA_MANIFEST_ID}" >> "$GITHUB_OUTPUT"
 
           DATA_VERSION=$(jq -r '."data-version"' "${{ env.PACKAGE_JSON_PATH }}")
-          echo "DATA_VERSION=${DATA_VERSION}" >> "$GITHUB_ENV"
+          echo "DATA_VERSION=${DATA_VERSION}" >> "$GITHUB_OUTPUT"
+
+          DATA_IMAGE_TAG="${DATA_MANIFEST_ID}.${DATA_VERSION}"
+          echo "DATA_IMAGE_TAG=${DATA_IMAGE_TAG}" >> "$GITHUB_OUTPUT"
 
-          DATA_IMAGE_TAG="${{ env.DATA_IMAGE_NAME }}:${DATA_MANIFEST_ID}-v${DATA_VERSION}"
-          echo "DATA_IMAGE_TAG=${DATA_IMAGE_TAG}" >> "$GITHUB_ENV"
+          DATA_IMAGE_PATH="ghcr.io/${{ env.NAMESPACE }}/${{ env.DATA_IMAGE_NAME }}:${DATA_IMAGE_TAG}"
+          echo "DATA_IMAGE_PATH=${DATA_IMAGE_PATH}" >> "$GITHUB_OUTPUT"
+
+      - name: login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
-      # TODO - check remote repository
       - name: check whether image exists
-        id: image-exists
+        id: check-container-repo
+        env:
+          # https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28#list-package-versions-for-a-package-owned-by-an-organization
+          ORG_PATH: '/orgs/${{ env.ORG_NAME }}/packages/container/${{ env.DATA_IMAGE_NAME }}/versions'
+          # https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28#list-package-versions-for-a-package-owned-by-the-authenticated-user
+          USER_PATH: '/user/packages/container/${{ env.DATA_IMAGE_NAME }}/versions'
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DATA_IMAGE_TAG: ${{ steps.get-ids-from-package-json.outputs.DATA_IMAGE_TAG }}
         run: |
-          echo "==> data image tag: ${{ env.DATA_IMAGE_TAG }}"
-          echo "IMAGE_EXISTS=false" >> "$GITHUB_ENV"
+          GH_PATH=$( [[ "${{ env.NAMESPACE }}" == "${{ env.ORG_NAME }}" ]] && echo "${{ env.ORG_PATH }}" || echo "${{ env.USER_PATH }}" )
```

Comment:
If the job is running in the Sage-Bionetworks repo, then check for the image in the organization namespace using the ORG_PATH. Otherwise, check for the image in the user's namespace using the USER_PATH.